### DISCUSSION TITLE
Transition: Add a delay prop

### DISCUSSION THREAD

**Discussion by gMichaudSpordle** [mantinedev/mantine#2067] [Feature requests]
Hi !  I've just started to use the mantine library and I absolutely love it. 

I was just wondering if it would be possible to add an option to the Transition component.

I want to stagger the transitions in a list by using the map index. However, there is no "delay" prop and I was wondering if it would be possible to add it.

Thank you for your time !

**Comment by rtivital** [comment]
Hi @gMichaudSpordle, thanks for kind words! I'll add enter and exit delays to Transition component in one of the next minor releases.

**Reply by geoguide** [reply]
Is this possible now? I don't really see in the docs how it would work.

**Comment by rtivital** [comment]
This feature will be available in 7.11.0 minor release with `enterDelay` and `exitDelay` props.
